utils/wrappers/aws/api_gateway.py

```diff
@@ -68,5 +69,13 @@ def wrapper(event: dict, context: LambdaContext, *args: Any, **kwargs: Any) -> d
                         'error': str(error)
                     })
                 }
+            except Exception as error:
+                return {
```

Comment:
Shouldn't we log the error? I guess we will return 500 and just suppress the error itself.

Reply:
Added logs.